token-lending/cli/src/main.rs

```diff
@@ -529,6 +538,15 @@ fn main() {
                         .required(false)
                         .help("Amount of liquidation bonus going to fee reciever: [0, 100]"),
                 )
+                .arg(
+                    Arg::with_name("protocol_take_rate")
```

Comment:
`update-reserve` needs to be updated to take this new arg

Reply:
Is this same snippet supposed to appear two times?

Reply:
there are 2 different functions so yeh 2nd spot needed a few other things to actually work